src/project.coffee

```diff
@@ -360,7 +360,6 @@ class Project extends Model
 
   addBuffer: (buffer, options={}) ->
     @addBufferAtIndex(buffer, @buffers.length, options)
-    @subscribeToBuffer(buffer)
```

Comment:
Apparently, we were subscribing to buffers twice, once here and once in `addBufferAtIndex`.